### ISSUE TITLE
Clipboard invoice is not detected.

### ISSUE THREAD

**Issue by roeierez** [breez/c-breez#327]
I am using this scneario:

1. Run c-breez wallet.
2. From my **Breez** wallet create invoice and copy it to clipboard.
3. Switch to c-breez and I don't get the prompt.

Invoice is only detected when I restart c-breez but not when switching from background. 
Same issue happens with node id in clipboard (keysend).

**Comment by ademar111190** [comment]
An update on this issue.

I did add mobile support to clipboard watcher https://github.com/leanflutter/clipboard_watcher/pull/8
In the process, I did discover important information; Android 10+ doesn't share the clipboard data copied from other apps with our app unless our app is in the foreground, which means the automatic recognition will not work on android 10+ when our app is in the background.
More info https://developer.android.com/about/versions/10/privacy/changes#clipboard-data

The good thing is we can copy when it is back to the foreground (has focus), so once that PR is accepted, I'll be able to add this library and get the clipboard data again without problems and without broken timer triggers.

@roeierez, you see the notification because every time we touch the clipboard to get the content, Android 10+ notifies the users with that notification. However, if the app still needs to be in focus (kind of messy android lifecycle), we ate getting null from the clipboard.

So let's wait for that PR to be merged and migrate our clipboard logic to use it and solves this and others issues I saw on clipboard flows.

Important info on iOS clipboard flow as well; it is necessary to get the user's permission otherwise, it will not work as well.

![meme 2023](https://user-images.githubusercontent.com/1225438/212903966-ce8e4d1a-5f41-4cc4-8b59-50f5392c2ba7.jpeg)
